Sources/AloeStackView/Views/StackViewCell.swift

```diff
@@ -41,6 +41,15 @@ open class StackViewCell: UIView {
   }
 
   // MARK: Open
+    
+  open override var isHidden: Bool {
```

Comment:
Could you give me some details on what this property override is for and why it is needed?

Reply:
hidden, remove, add There is a phenomenon that seperator views overlap when animation occurs. We have added this code to improve it.

Reply:
Ah yeah I was wondering if that was the reason. This looks like it could be a good fix.

Could you please put this in its own PR so we can review it independently though? I’d like to see how the animation looks after this fix, but there’s no need to hold up the rest of the PR for it.

Let me know if that is ok!

Reply:
I will give you a PR for this work :)